### DISCUSSION TITLE
关于test路径错误的问题

### DISCUSSION THREAD

**Discussion by wenhaixi** [STVIR/pysot#511] [Q&A]
在安装环境完成后，我这里是用的pycharm，参数设置为--snapshot
../experiments/siamrpn_alex_dwxcorr/model.pth
--dataset
OTB100
--config
../experiments/siamrpn_alex_dwxcorr/config.yaml
运行后显示AssertionError: C:\Users\Administrator\Desktop\python\pysot\tools\Basketball\img\0001.jpg
通过debug发现，在video.py下，self.img_names = [os.path.join(os.path.abspath(root), os.path.abspath(x)) for x in img_names ]这行代码进行拼接路径，然而我通过os.path.abspath(self.img_names[0])发现路径根本没有变化， 仍然是..\python\pysot\tools\Basketball\img\0001.jpg。

**Comment by wenhaixi** [comment]
解决了，提供的代码有错误，拼接路径不可以用os.path.abspath(x)，这里是绝对路径，拼接不起来，只需要x就可以了。

**Reply by ZhiyuanChen** [reply]
hmmmm这是否是已经在https://github.com/STVIR/pysot/pull/459修复？
